packer/plugin-getter/plugins.go

```diff
@@ -0,0 +1,421 @@
+package plugingetter
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/packer-plugin-sdk/tmp"
+	"github.com/hashicorp/packer/hcl2template/addrs"
+)
+
+type Requirements []*Requirement
+
+// Requirement describes a required plugin and how it is installed. Usually a list
+// of required plugins is generated from a config file. From it we check what
+// is actually installed and what needs to happen to get in the desired state.
+type Requirement struct {
+	// How the user named the plugin
```

Comment:
```suggestion
	// name to use when using the plugin in the Packer HCL config
```

Reply:
I made it a bit longer:
https://github.com/hashicorp/packer/blob/d8a93c61fad6d0b2d2539b1325c2803fea2de30d/packer/plugin-getter/plugins.go#L25-L29

I hope it's good !